src/models/TranslationKey/createTranslationKey/index.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import TranslationKey from '@models/TranslationKey'
+import type { ValidationError } from '@types/ValidationError'
+import validateString from '@helpers/validateString'
+import Project from '@models/Project'
+
+const validateKey = validateString({
+  type: 'translation key',
+  minLength: 1,
+  maxLength: 55
+})
+
+export type AddTranslationKeyToProjectArgs = {
+  key: string,
+  projectId: string,
+  ownerId: string
+}
+
+type AddTranslationKeyToProjectResponse = {
+  project?: Project,
+  errors: Array<ValidationError>
+}
+
+const createTranslationKey = async ({
+  key,
+  projectId,
+  ownerId
+}: AddTranslationKeyToProjectArgs): Promise<
+  AddTranslationKeyToProjectResponse
+> => {
+  const errors: Array<ValidationError> = []
+  const keyValidation = validateKey(key)
+
+  //  First some authorization stuff
+  const project: Project = await Project.query()
+    .findById(projectId)
+    .eager('translationKeys')
+
+  if (!project || (project && project.ownerId !== ownerId)) {
```

Comment:
It might be worth to extract `(project && project.ownerId !== ownerId)` to a `project.hasOwner(ownerId)` method?